elasticsearch/examples/hot-cold/Makefile

```
@@ -0,0 +1,15 @@
+default: test
+
+include ../../../helpers/examples.mk
+
+RELEASE := helm-es-hot-cold
+
+install:
+	helm init --client-only
+	helm dependency update
+	helm upgrade --wait --timeout=600 --install $(RELEASE) .
```

Can you add a variable `TIMEOUT := 1200s` like we are doing in the other Makefiles now
```suggestion
	helm upgrade --wait --timeout=$(TIMEOUT) --install $(RELEASE) .
```